enhancements/two-node-fencing/tnf.md

```diff
@@ -376,30 +384,42 @@ platform:
   baremetal:
     fencingCredentials:
       bmc:
-          address: ipmi://<out_of_band_ip>
-          username: <user>
+          address: https://<redfish-api-url>
+          username: <username>
           password: <password>
     apiVIPs:
       - <api_ip>
     ingressVIPs:
       - <wildcard_ip>
     hosts:
-      - name: openshift-master-0
+      - name: openshift-cp-0
         role: master
         bmc:
           address: ipmi://<out_of_band_ip>
-          username: <user>
+          username: <username>
           password: <password>
-      - name: <openshift_master_1>
+      - name: openshift-cp-1
         role: master
         bmc:
           address: ipmi://<out_of_band_ip>
-          username: <user>
+          username: <username>
           password: <password>
 pullSecret: ''
 sshKey: ''
 ```
 
+##### Why don't we reuse the existing APIs in the Baremetal platform?
+Reusing the existing APIs tightly couples separate outcomes that are important to distinguish for the end user.
+
+The `fencingCredentials` API exists for deploying TNF and is *only valid* when TNF is being deployed. Failure to define this section when using TNF will result in the installer throwing an error.
+
+The `hosts` API exists for configuring the BareMetalHost entries that will be managed by the Bare Metal Operator. The only caveat to this is that when TNF is deployed, we must also scan this list to
+ensure that control plane nodes are not included, and provide an error message explaining that BMC credentials for the control plane hosts shouldn't be provided to ironic in a TNF deployment. If we
```

Comment:
I think this could be simplified.
Maybe it's good enough to make sure that Bare Metal Operator `host` section is mutually exclusive with TNF `fencingCradentials` section.
IIUC only verifying CP nodes in the BMC, is an indication that we allow `Bare Metal Operator` to run on a TNF cluster which we want to avoid.

Reply:
I think you're right. Since we don't support compute nodes for now, we could look at revisiting this restriction down the line, but think for our purposes, mutual exclusivity is the simplest thing to understand from a user perspective. I'll update this section!

Reply:
Updated! Let me know what you think!